.github/workflows/test.yml

```diff
@@ -11,7 +11,11 @@ jobs:
       matrix:
         os: ["ubuntu-latest", "windows-latest", "macos-latest"]
         py: ["3.14-dev", "3.13", "3.12", "3.11", "3.10", "3.9"]
-
+        exclude:
+          - os: window-11-arm
+            py: "3.9"
+          - os: window-11-arm
+            py: "3.10"
```

Comment:
do you want add them into `wheel.yml`?

Reply:
Please look again at the diff of the two files being edited in this PR.

You are adding windows-11-arm to excludes, while window-11-arm is not included in the matrix of this file.
You probably forgot to add windows-11-arm to the matrix of this file, or you intended to add excludes to the other file, not this one.